sdk/ruby/lib/chain/transaction.rb

```diff
@@ -85,16 +88,17 @@ def submit(template)
         ) { |item| SubmitResponse.new(item) }
       end
 
-      # @param [Array<Template>] templates
-      # @return [Array<SubmitResponse>]
+      # Submit multiple signed transactions to the blockchain.
+      # @param [Array<Template>] templates Array of signed transaction templates.
+      # @return [BatchResponse<SubmitResponse>]
       def submit_batch(templates)
         client.conn.batch_request(
           'submit-transaction',
           {transactions: templates}
         ) { |item| SubmitResponse.new(item) }
       end
 
-      # @param [Hash] query
+      # @param [QueryOpts || Hash] query
```

Comment:
Transaction queries have a few additional options that should be exposed to the user:

- `timeout`
- `start_time`
- `end_time`

For details, see https://github.com/chain/chain/blob/main/docs/internal/api-spec.swagger.yml#L510